tools/get_average_calls_per_user.js

```diff
@@ -0,0 +1,46 @@
+var conf = require('../loop/config').conf;
+var async = require('async');
+var redis = require("redis");
+
+var storage = conf.get("storage");
+
+if (storage.engine === "redis") {
+  var options = storage.settings;
+  var client = redis.createClient(
+    options.port,
+    options.host,
+    options.options
+  );
+  if (options.db) client.select(options.db);
+
+  var multi = client.multi();
+  multi.eval("return #redis.pcall('keys', 'userUrls.*')", 0);
```

Comment:
It is some lua code that let you get the number of keys matching the pattern on server side.